pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -521,7 +521,6 @@ func (h *NetworkUtilsHandler) DisableTXOffloadChecksum(ifaceName string) error {
 }
 
 // Allow mocking for tests
-var SetupPodNetworkPhase1 = SetupNetworkInterfacesPhase1
 var SetupPodNetworkPhase2 = SetupNetworkInterfacesPhase2
```

Comment:
I think the same can be done for `SetupPodNetworkPhase2`.
Although there is a test actually modifying it, the test subject doesn't use `SetupPodNetworkPhase2` it is directly invoking `SetupNetworkInterfacesPhase2`.